packages/server/src/aws/index.ts

```diff
@@ -4,7 +4,7 @@ import {
   Webhooks,
   WebhookData,
   WebhookHandler,
-} from '@sane-shopify/types'
+} from '@six-socks-studio/sane-shopify-types'
```

Comment:
Once this is changed back to `@sane-shopify` it should be good to go! Does `@sane-shopify/types` need to be updated or was this just something you did to get your local version running?